lib/message_manager.rb

```diff
@@ -0,0 +1,29 @@
+require_relative './database_connection'
+
+class Message
+
+  def self.post_peep(content:)
+    result = DatabaseConnection.query("INSERT INTO messages \
+      (content, timestamp) VALUES('#{content}', '#{Time.now}') \
+      RETURNING id, content, timestamp")
+    Message.new(id: result[0]['id'], content: result[0]['content'], \
+      timestamp: result[0]['timestamp'])
+  end
+
+  def self.show_peeps
+    result = DatabaseConnection.query("SELECT * FROM messages \
+      ORDER BY timestamp DESC")
+    result.map do |peep|
+      Message.new(id: peep['id'], content: peep['content'], \
+        timestamp: peep['timestamp'])
+    end
+  end
+
+  attr_reader :id, :content, :timestamp
+
+  def initialize(id:, content:, timestamp:)
```

Comment:
You can have the database automatically apply timestamps rather than passing them in with you program objects - when creating a table you can use:

```
CREATE TABLE peeps (
  id SERIAL PRIMARY KEY,
  date_added timestamp DEFAULT CURRENT_TIMESTAMP,
  content varchar(240)
);
```
CURRENT_TIMESTAMP is a PostgreSQL constant that holds the current time.